Specs/DateTime/English/MergedParser.json

```diff
@@ -1322,12 +1318,179 @@
   ]
 }
 ,
+{
+  "Input": "I'll leave after or equal to 1/1/2016",

```

Comment:
Please change the sentence to "give me all open spots with dates after or equal to 1/1/2016"